utils/mediaupload.js

```diff
@@ -1,7 +1,12 @@
 /**
  * External Dependencies
  */
-import { compact, forEach, get, noop, startsWith } from 'lodash';
+import { compact, forEach, get, includes, noop, startsWith } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '../node_modules/@wordpress/i18n';
```

Comment:
With all the changes introduced in here (translations, WordPress specific settings), I don't think this file makes sense in `utils` module anymore. Should we move it to the `editor` module? and keep it as an internal helper for `editorMediaUpload`? By the way, is it used somewhere else?

Reply:
I agree with moving mediaUpload to the editor module, and given that mediaUpload is not used anywhere besides editorMediaUpload probably the two functions can be consolidated together.
If doing this we should leave mediaUpload as it was in utils for 2 versions with a deprecation message.